modules/apps/text-localizer/text-localizer-taglib/src/main/java/com/liferay/text/localizer/taglib/internal/address/JPAddressTextLocalizer.java

```diff
@@ -37,23 +35,20 @@ public class JPAddressTextLocalizer implements AddressTextLocalizer {
 	public String format(Address address) {
 		StringBundler sb = new StringBundler(13);
 
-		Optional<String> countryNameOptional =
-			AddressUtil.getCountryNameOptional(address);
+		String countryName = AddressUtil.getCountryName(address);
 
-		countryNameOptional.ifPresent(
-			countryName -> {
-				sb.append(html.escape(countryName));
-				sb.append(StringPool.NEW_LINE);
-			});
+		if (countryName != null) {
+			sb.append(html.escape(countryName));
+			sb.append(StringPool.NEW_LINE);
+		}
 
 		Address escapedAddress = address.toEscapedModel();
 
 		String city = escapedAddress.getCity();
 
 		boolean hasCity = Validator.isNotNull(city);
 
-		Optional<String> regionNameOptional = AddressUtil.getRegionNameOptional(
-			address);
+		String regionName = AddressUtil.getRegionName(address);
```

Comment:
@JanisZhang Could you SF it to move this variable down before line 61? It's not used until that line but declared 10 lines earlier.

Reply:
sure Thanks!